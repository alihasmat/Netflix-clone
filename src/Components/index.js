export { default as Accordian } from "./Accordian/Accordian";
export { default as Card } from "./Card/Card";
export { default as Feature } from "./Feature/Feature";
export { default as Footer } from "./Footer/Footer";
export { default as Form } from "./Form/Form";
export { default as Header } from "./Header/Header";
export { default as Jumbotron } from "./Jumbotron/Jumbotron";
export { default as Loading } from "./Loading/Loading";
export { default as Player } from "./Player/Player";
export { default as Profiles } from "./Profiles/Profiles";
export { default as Search } from "./Search/Search";
