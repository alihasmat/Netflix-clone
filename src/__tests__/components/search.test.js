import { render, fireEvent } from "@testing-library/react";
import { Search } from "../../Components";

describe("<Search />", () => {
  it(`renders <Search />`, () => {
    const { container, getByText, getByPlaceholderText } = render(
      <Search>
        <Search.Text>
          Ready to watch? Enter your email to create or restart your membership.
        </Search.Text>
        <Search.Form>
          <Search.Input placeholder="Email Address" />
          <Search.Button>Get Started</Search.Button>
        </Search.Form>
      </Search>
    );
    expect(
      getByText(
        "Ready to watch? Enter your email to create or restart your membership."
      )
    ).toBeTruthy();
    expect(getByPlaceholderText("Email Address")).toBeTruthy();
    expect(getByText("Get Started")).toBeTruthy();
    expect(container.firstChild).toMatchSnapshot();
  });
});
