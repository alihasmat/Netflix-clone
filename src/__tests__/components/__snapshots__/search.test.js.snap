// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Search /> renders <Search /> 1`] = `
<div
  class="style__Container-sc-2194aj-0 ffykDg"
>
  <p
    class="style__Text-sc-2194aj-1 eCvxxC"
  >
    Ready to watch? Enter your email to create or restart your membership.
  </p>
  <div
    class="style__Form-sc-2194aj-2 cuDPLU"
  >
    <input
      class="style__Input-sc-2194aj-3 kJZYId"
      placeholder="Email Address"
    />
    <button
      class="style__Button-sc-2194aj-4 cYGyku"
    >
      Get Started
       
      <img
        alt="get started"
        src="/images/icons/chevron-right.png"
      />
    </button>
  </div>
</div>
`;
